### PR TITLE
Add test cases to cover podman volume

### PR DESCRIPTION
**Pull request by yuhuijiang**
Add test cases to cover below podman volume subcommand:
  create
  ls
  inspect
  rm
  prune

### CODE REVIEW

File: test/apiv2/30-volumes.at [hunk 1]
```diff
@@ -3,12 +3,75 @@
...
+# Negative test
+t DELETE libpod/volumes/foo1 404 \
+    .cause="no such volume" \
+    .message~.* \
+    .response=404    
+
+## Prune volumes
+
+t POST libpod/volumes/prune "" 200
+#After prune volumes, there shoudl be no volume existing
```
Comment (TomSweeneyRedHat) [inline test/apiv2/30-volumes.at:73]: nit of a nit
```suggestion
#After prune volumes, there should be no volume existing
```

File: test/apiv2/30-volumes.at [hunk 2]
```diff
@@ -3,12 +3,75 @@
...
-#
-# FIXME: endpoints seem to be unimplemented, return 404
-#
-if false; then
-t GET  libpod/volumes/json            200 null
-t POST libpod/volumes/create name=foo 201
-fi
+## create volume
+
+voluempath=$(curl -s http://$HOST:$PORT/v1.40/libpod/info | jq -r ".store.volumePath")
```
Comment (TomSweeneyRedHat) [inline test/apiv2/30-volumes.at:8]: Did you mean to make this "voluempath" instead of "volumepath"?

**Review by TomSweeneyRedHat — COMMENTED** [review]
test issues and a couple nits/questions, otherwise LGTM

File: test/apiv2/30-volumes.at (hunk 2 above)
Comment (yuhuijiang) [reply test/apiv2/30-volumes.at]: It's a typo,  I have corrected it. Thanks.

**Comment by rhatdan** [comment]
@yuhuijiang You need to cleanup whitespace.

**Pull request closed without merging**